sdk-lib/src/main/java/cash/z/ecc/android/sdk/block/CompactBlockProcessor.kt

```diff
@@ -574,11 +574,14 @@ class CompactBlockProcessor internal constructor(
                 @Suppress("TooGenericExceptionCaught")
                 try {
                     retryUpTo(3) {
+                        val tAddresses = rustBackend.listTransparentReceivers(account.value)
```

Comment:
I remember that recently we had changes in how the transparent addresses were derived, as part of supporting unified addresses.

Does this method get both the old and the new transparent address?

Reply:
What I can see is that to get a single transparent address, which we used before these changes -  `getTransparentReceiverForUnifiedAddress()`, we got a transparent address for a unified address from the Rust layer. Now we use `listTransparentReceivers()` to get all transparent addresses from the Rust layer, which implementation seems different to me, but I might be wrong. So please @str4d, could you also take a quick look at this thought? Thank you.